### PR TITLE
Update mdc snackbar, progress bar, spinner

### PR DESCRIPTION
**Pull request by bastianjoel**
part of #2934

### CODE REVIEW

**Review by Elblinator — CHANGES_REQUESTED** [review]
in darkmode: progress bar looks always completely filled

in darkmode: in a snackbar can the text color be as dark as is was before?

**Comment by jsangmeister** [comment]
@bastianjoel please resolve conflicts

**Comment by Elblinator** [comment]
Is this PR really ready for review= Is my CR fixed? 
Please assign me accordingly

**Comment by bastianjoel** [comment]
> in darkmode: in a snackbar can the text color be as dark as is was before?

I would just stick with the default there.

**Review by Elblinator — APPROVED** [review]

File: client/src/assets/styles/theme-styles.scss [hunk 1]
```diff
@@ -8,19 +8,27 @@
...
+    .mat-mdc-progress-bar {
+        --mdc-linear-progress-track-color: var(--theme-primary-100);
+    }
+}
+
 @include mat.all-component-themes($openslides-light-theme);
 
 .openslides-light-theme {
     /*TODO(mdc-migration): Remove all-legacy-component-themes once all legacy components are migrated */
     @include mat.all-legacy-component-themes($openslides-light-theme);
```
[the remark this answers is not in the document]
Comment (bastianjoel) [reply client/src/assets/styles/theme-styles.scss:21]: We need to wait until all components are migrated.

**Pull request merged**